src/components/TokenClaimDialog.tsx

```diff
@@ -77,105 +86,124 @@ export default function TokenClaimModal(): ReactElement {
   }, [chainId, rewardBalances])
 
   return (
-    <div data-testid="tknClaimContainer" className={styles.container}>
-      <div className={styles.gradient}></div>
-      <div className={styles.logoWrapper}>
-        <div className={styles.logo}>
-          <img src={logo} />
+    <Dialog
+      open={open}
+      scroll="body"
+      onClose={onClose}
+      sx={{ overflow: "hidden" }}
+      data-testid="tokenClaimDialog"
+    >
+      {/* TODO: Remove this button after update the modal */}
+      <IconButton
+        disableRipple
```

Comment:
lol, is there a way to globally disable ripples?

Reply:
There is .
I did this on the another branch. But it is in progress. :)

Reply:
We can. 
I already added that on another branch and that branch is still in progress. Let me remove that props when we update modal again.